pkg/virt-controller/watch/vmi.go

```diff
@@ -345,7 +349,22 @@ func (c *VMIController) execute(key string) error {
 	}
 
 	return nil
+}
+
+func (c *VMIController) syncPodAnnotations(updatedPod *k8sv1.Pod) error {
+	patchBytes, err := prepareAnnotationsPatch(updatedPod.Annotations)
+	if err != nil {
+		return fmt.Errorf("failed to prepare patch pod: %v", err)
```

Comment:
I think this info needs to be added in `prepareAnnotationsPatch`.

Reply:
moved